pydoctor/templates/apidocs.css

```diff
@@ -108,6 +108,7 @@ ul ul ul ul ul ul ul {
     padding: 5px;
 }
 
+
```

Comment:
This extra empty line isn't consistent with the surrounding rules (a leftover of reverted edits?).